pillar.example

```diff
@@ -19,3 +19,38 @@ docker:
     all: true # true (Default): Remove all unused images, translates to `docker system prune --force --all`
               # false: (Remove only dangling images, translates to `docker system prune --force`
     volumes: false # Remove unused volumes as well. Default: false
+
+
+# Registry credentials for the docker.login state (if used)
+#
+# Note that multiple dictionaries, and multiple registries per dictionary, can be specified:
+#
+
+docker-registries:
+  'https://docker-registry.work.com':
+    username: jrabbit
+    password: s3cr3t
+
+my-private-docker-registries:
+  'https://docker-A.jack-rabbit.com':
+    username: jack
+    password: c4rr0t
+  'https://docker-B.jack-rabbit.com':
+    username: jack
+    password: c4rr0t
+
+
+# Configuration for the docker.machine substate
+#
+# See README on how to obtain the contents of the key and certificate files
+#
+docker-machine:
+  version: v0.16.2-gitlab.12
```

Comment:
I think there should be a hint where to get that (and the most recent) version number from. I suppose their repository? Maybe they also support something like `HEAD` or a branch like `main`? If so, that should be also documented.

Reply:
- bumped to latest releasse
- docs explicitly suggest to fetch a designated release, and I haven't found anything about fetching `main`/`HEAD`

Reply:
Bit cumbersome to be up to date here. I don't know any internal solution - it's possible to query the Github API for the latest version, but that's outside of the scope of this formula.